src/client/fsharp/Oak/View.fs

```diff
@@ -201,12 +201,22 @@ let limitTreeByNodes =
         limitTree allowedNodes n)
 
 let createGraph =
+    // copied from variables.sass
+    let colors = {|
```

Comment:
This is mind-blowingly clever to use an anonymous record at this location.
Pure genius 🤯